### ISSUE TITLE
Dependencies in meta.yaml out of date?

### ISSUE THREAD

**Issue by xylar** [E3SM-Project/e3sm_to_cmip#12]
I think some of the dependencies aren't consistent with the latest changes:
https://github.com/E3SM-Project/e3sm_to_cmip/blob/master/conda/meta.yaml

Specifically, I think `tqdm` isn't needed but `progressbar2`, `pyyaml` and `scipy` are.  Keeping this updated (as well as adding the dependencies to `README.md` and/or `setup.py`) would help me keep track of them.  Otherwise, I have to go through some pretty painful trial and error.

For the MPAS handlers, I'll need `xarray`, `dask` and `netcdf4` added as well.

**Comment by xylar** [comment]
Here's the command I'm using to create the environment I'm using on Anvil to test:
```
conda create -n e3sm_to_cmip -c conda-forge nco cmor cdutil cdms2 progressbar2 pyyaml xarray \
    netcdf4 dask scipy
```

**Comment by xylar** [comment]
I've mostly addressed this in #11, though I didn't try to edit `setup.py`.
